Examples/Scripts/PreFreeSurferPipelineBatch.sh

```diff
@@ -283,7 +292,7 @@ main()
 		# volume or "NONE" if not used
 		PhaseInputName="${StudyFolder}/${Subject}/unprocessed/3T/T1w_MPR1/${Subject}_3T_FieldMap_Phase.nii.gz"
 
-		# The TE variable should be set to 2.46ms for 3T scanner, 1.02ms for 7T
+		# The DeltaTE (echo time difference) variable should be set to 2.46ms for 3T scanner, 1.02ms for 7T
```

Comment:
If we wanted to, we could clarify that these example numbers are for HCP YA data.

Reply:
Yes, it sound like a good opportunity

Reply:
TE was repalced by DeltaTE in PreFreeSurferPipelineBatch.sh